extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -108,6 +114,45 @@ describes.realWin(
       expect(text).to.equal('Hello Joe');
     });
 
+    it('renders from amp-script', async () => {
+      const ampScript = html`
+        <amp-script id="dataFunctions" script="local-script" nodom></amp-script>
+      `;
+      const fetchScript = html`
+          <script id="local-script" type="text/plain" target="amp-script">
+            function getRemoteData() {
+              return fetch('https://example.com/data.json')
+                  .then((resp) => resp.json());
+            }
+            exportFunction('getRemoteData', getRemoteData);
+          </script>
+        </div>
```

Comment:
This might prevent prettier from formatting properly.

```suggestion
```

Reply:
Done